elements/pl-drawing/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "controller": "pl-drawing.py",
+    "dependencies": {
+        "coreScripts": [
+            "sylvester.js",
+            "sha1.js",
+            "underscore.js"
+        ],
+        "elementScripts": [
+            "fabric.require.js",
```

Comment:
Is it possible to install `fabric` with `npm`, rather than including a packaged source distribution directly?